src/sentry/api/serializers/models/organization.py

```diff
@@ -598,6 +599,7 @@ def serialize(  # type: ignore[explicit-override, override]
         if access.role is not None:
             context["role"] = access.role  # Deprecated
             context["orgRole"] = access.role
+        context["requiresSso"] = access.requires_sso
```

Comment:
Is there another way we can make this check without having to update the org serializer? I'm sure we must know somewhere on the frontend if the org requires SSO no?

Reply:
Chatted with @RyanSkonnord about this. We really don't expose this anywhere to the UI for the most part. I think adding this here is fine (despite it being just another field on an organization).

Reply:
Some notes on my investigation, for the record:

The source of truth for `access.requires_sso` is `query_sso_state` at https://github.com/getsentry/sentry/blob/master/src/sentry/auth/services/access/service.py#L60, which inspects the associated `AuthProvider` object. The flag is generally available through an `Access` object and isn’t associated directly with the org model. Given that the org serializer is already adding few other `Access` attributes to the API output (right above here), there seems to be precedent. However, I did have the same initial intuition as @evanpurkhiser — it's surprising that it wasn’t already visible somehow.